common/scala/src/main/scala/whisk/core/WhiskConfig.scala

```diff
@@ -129,12 +129,16 @@ object WhiskConfig extends Logging {
                 else null
             } else null
 
-        val dir = sys.props.get("user.dir")
```

Comment:
are the changes below accomplished more simply with just this one line change?
`val dir = sys.props.get("user.dir") orElse sys.env.get("OPENWHISK_HOME")`